docker-services.yml

```diff
@@ -52,8 +52,8 @@ services:
   db:
     image: postgres:12.4
     environment:
-      - "POSTGRES_USER=test"
-      - "POSTGRES_PASSWORD=psw"
+      - "POSTGRES_USER=invenio"
```

Comment:
This should not probably be committed? The default config in config.py is:
`SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2://test:psw@localhost/ils"`